### PR TITLE
feat(samples): add integrity verification to Cloud KMS crypto samples

### LINKED ISSUE #408: [kms] Add integrity verification to Cloud KMS's node crypto samples
Enahancing Cloud KMS's samples to document how to perform end-to-end integrity verification.

### PR DESCRIPTION
**Pull request by iamtamjam**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-kms/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #408 🦕

### CODE REVIEW

**Comment by iamtamjam** [comment]
@googleapis/yoshi-nodejs Hello reviewers. I'll be updating several files with changes similar to those in samples/encryptSymmetric.js. Can you please provide preliminary feedback to speed up the process? 

Also, despite adding a rule to allow the use of external `fast-crc32c` module in `package.json`, I'm still seeing the following Lint error: `"fast-crc32c" is extraneous  node/no-extraneous-require`.  Can you please advise on how to resolve this?

**Comment by iamtamjam** [comment]
@bcoe Hi bcoe! Hope you're the right person to ping. I'm awaiting review on the PR. Are you able to help with that? 

I also need help with a persistent lint error: `"fast-crc32c" is extraneous node/no-extraneous-require`.
I followed instructions [here](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-extraneous-require.md) and added a `rules` section to `package.json` with `fast-crc32c` module in the `allowModules` section, but to no avail.

Do I need to do something different? 

Thanks in advance!
Tamara

File: package.json [hunk 1]
```diff
@@ -45,9 +45,17 @@
     "api-documenter": "api-documenter yaml --input-folder=temp"
   },
   "dependencies": {
+    "fast-crc32c": "^2.0.0",
```
Comment (bcoe) [inline package.json:48]: this dependency should be added to `samples/package.json`, vs., `package.json`, at which point if you run:

```
rm -rf node_modules
rm package-lock.json
rm -rf samples/package-lock.json
rm -rf samples/node_modules
npm run i
npm run lint
```

From the root directory of the library, I believe you'll stop seeing the `no-extraneous-require` linting issues.

File: package.json [hunk 2]
```diff
@@ -45,9 +45,17 @@
...
     "google-gax": "^2.9.2"
   },
+  "rules": {
+    "node/no-extraneous-require": ["error", {
```
Comment (bcoe) [inline package.json:52]: this shouldn't be needed once you move the dependency to samples.

**Review by bcoe — CHANGES_REQUESTED** [review]
Sorry for the slow review, I was away last week.

This looks reasonable to me, once you address the notes related to extraneous require.

File: package.json (hunk 1 above)
Comment (iamtamjam) [reply package.json]: Done.

File: package.json (hunk 2 above)
Comment (iamtamjam) [reply package.json]: Done. It did resolve the lint issue. Thanks!

**Comment by iamtamjam** [comment]
> low review, I was away last week.

**Comment by iamtamjam** [comment]
Can you please take another look and merge?

**Comment by iamtamjam** [comment]
Ah.. accidentally closed this PR. @bcoe , your proposed changes have resolved the lint issue. Can you please merge this PR? Thanks in advance!

**Comment by iamtamjam** [comment]
@bcoe I see you updated this branch. Just to confirm: this PR is ready to be merged, right?

**Review by bcoe — APPROVED** [review]

**Comment by bcoe** [comment]
@iamtamjam sorry for the delay, merged 👍

**Comment by iamtamjam** [comment]
Thanks for reviewing and merging, @bcoe !

**Pull request merged**